tests/post/features/example_solution.feature

```diff
@@ -0,0 +1,31 @@
+@local @solution
+Feature: Example solution
+    Scenario: Deploy Solution
+        Given the Kubernetes API is available
+        And no Solution 'example-solution' is imported
+        And no Solution environment 'example-environment' is available
+        And the Solution Configuration file is absent
+        When we import a Solution archive '/var/tmp/example-solution.iso'
+        Then Solution archive 'example-solution' is imported correctly
+        When I activate Solution 'example-solution' with version '0.1.0-dev'
+        Then Solution 'example-solution' is activated
+        When I create a solution environment 'example-environment' for Solution 'example-solution'
+        Then solution environment 'example-environment' is available
+        When we remove Taints on Bootstrap node before deployment
+        And we deploy Solution 'example-solution' in environment 'example-environment' with version '0.1.0-dev'
+        Then the 'pods' list should not be empty in the 'example-environment' namespace
+        When we check the available replicas 'status.available_replicas' for 'example-solution-operator' deployment in environment 'example-environment'
+        Then we have '1' available 'status.available_replicas' for Solution 'example-solution-operator' Deployment in environment 'example-environment'
+        When we check the available replicas 'status.available_replicas' for 'example-solution-ui' deployment in environment 'example-environment'
+        Then we have '1' available 'status.available_replicas' for Solution 'example-solution-ui' Deployment in environment 'example-environment'
+        When we deactivate Solution 'example-solution'
+        And we delete Solution 'example-solution' in environment 'example-environment'
+        And we delete Solution environment 'example-environment'
+        And we unimport Solution archive '/var/tmp/example-solution.iso'
+        And we delete the Solution Configuration file
+        Then we have no Solution 'example-solution' archive mounted
+        And we have no Solution environment 'example-environment'
+        And we have no activated Solution 'example-solution'
+        And we have no Solution Configuration file present
+        And we have no Solution deployment 'example-solution-operator' in environment 'example-environment'
+        And we have no Solution deployment 'example-solution-ui' in environment 'example-environment'
```

Comment:
These 2 will always be true since you're testing `And we have no Solution environment 'example-environment'` just before.